core/jobs/registry.py

```diff
@@ -57,6 +57,7 @@
 from core.jobs.batch_jobs import user_stats_computation_jobs         # pylint: disable=unused-import  # isort: skip
 from core.jobs.batch_jobs import math_interactions_audit_jobs        # pylint: disable=unused-import  # isort: skip
 from core.jobs.batch_jobs import exp_title_validation_jobs           # pylint: disable=unused-import  # isort: skip
+from core.jobs.batch_jobs import exp_model_validation_jobs           # pylint: disable=unused-import  # isort: skip
```

Comment:
Can we rather rename `exp_title_validation_jobs` to `exp_validation_jobs` and just have one file?

Reply:
Done!